forms/manager.py

```diff
@@ -0,0 +1,6 @@
+from django.db import models
```

Comment:
Why not in `modelutils.py`?

Reply:
@kilemensi  wanted to have a better separation but looking at it again it makes sense to have it inside this file. I've made the changes.